website/static/examples/show.html

```diff
@@ -0,0 +1,54 @@
+<head>
+  <meta name="monetization" content="$twitter.xrptipbot.com/Interledger">
```

Comment:
Should this use the coil payment pointer, not sure this one actually works...

Reply:
https://twitter.com/Interledger it's certainly a legitimate twitter account. It feels weird to make the examples on webmonetization.org pay to Coil. If there were a verified charity I'd be good with using it in examples

Reply:
I'll also include a note that this should be changed to your own payment pointer

Reply:
I agree that it ideally should not be coil, can we make both go to interledger then? I noticed the other one went to Coil.